### PR TITLE
DOC-4359: Update remote edit documentation with details for linux OS

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/Manage-Documents.rst [hunk 1]
```diff
@@ -425,9 +417,9 @@ requirements before using this feature.
 
 
 .. note:: For all OSs/browsers, it is recommended you install and make sure
-			Java Applet enabled. This is required for opening Non-MS Office
-			files. You can visit http://javatester.org/ to make sure Java Plugin
-			already installed on your browser.
+		  Java Applet enabled. This is required for opening Non-MS Office
```
Comment (ahamdi) [inline docs/Manage-Documents.rst]: I am not sure if this is still required

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: docs/Manage-Documents.rst (hunk 1 above)
Comment (mzorai) [reply docs/Manage-Documents.rst]: OK, should I remove this note then?

Comment (ahamdi) [reply docs/Manage-Documents.rst]: @hassenelkamel-exo Could you confirm please?

Comment (hassenelkamel-exo) [reply docs/Manage-Documents.rst]: I m agree with you @ahamdi , java applet is not required

Comment (mzorai) [reply docs/Manage-Documents.rst]: OK so I remove this note from documentation

**Review by hassenelkamel-exo — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**